### ISSUE TITLE
emb-w-07 - Application Window

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#8]
emb-w-07	Application Window	Yes	3-Medium	

Insufficient color contrast for "Write Files", "Stop Editing", "Apply Changes" buttons and table information/controls.	

Ensure color contrast of at least 4.5:1 for text.

**Comment by bertramlyons** [comment]
In progress currently.

**Comment by bertramlyons** [comment]
Completed and merged in version 1.3.0.
